test/Tests/src/Data/Statistics_Spec.enso

```diff
@@ -168,11 +168,11 @@ spec =
 
         Test.specify "should fail with Incomparable_Values on custom incomparable type" <|
             no_ord_set.compute . should_equal 3
-            no_ord_set.compute Statistic.Minimum . should_fail_with Incomparable_Values
-            no_ord_set.compute Statistic.Maximum . should_fail_with Incomparable_Values
+            no_ord_set.compute Statistic.Minimum . should_fail_with Incomparable_Values.Error
+            no_ord_set.compute Statistic.Maximum . should_fail_with Incomparable_Values.Error
```

Comment:
I already placed it in a different test. Here, I don't think we know what should be `left` and what should be `right`. It's up to the implementation of `compute` method which values it will compare first.